src/specs/av1_isobmff/av1_isobmff.cpp

```diff
@@ -510,9 +511,8 @@ const SpecDesc specAv1ISOBMFF = {
         out->covered();
       } },
     { "Section 2.2.4\n"
-      "The width and height in the TrackHeaderBox SHOULD equal, respectively, the maximum\n"
-      "RenderWidth, called MaxRenderWidth, and the maximum RenderHeight, called\n"
-      "MaxRenderHeight, of all the frames associated with this sample entry",
+      "The width and height in the TrackHeaderBox SHOULD equal, respectively, the maximum RenderWidth, called MaxRenderWidth, and the maximum RenderHeight, called MaxRenderHeight, of all the frames associated with this sample entry.",
```

Comment:
"The width and height in the TrackHeaderBox SHOULD be equal to MaxRenderWidth and MaxRenderHeight, respectively.",
  "assert-b8425a84",